lib/counter_web/live/counter_live.ex

```diff
@@ -0,0 +1,28 @@
+defmodule CounterWeb.CounterLive do
+  use Phoenix.LiveView
+
+  def mount(_session, _, socket) do
+    socket = assign(socket, :count, 0)
+    {:ok, socket}
+  end
+
+  def render(assigns) do
+    ~L"""
+    <h1>Count <%= @count %> </h1>
+    <button phx-click="increment">+</button>
+    <button phx-click="decrement">-</button>
+    """
+  end
+
+  def handle_event("increment", _value, socket) do
```

Comment:
@JuanMaGiraldo what a handle_event does?

Reply:
It is an implementation of a callback, in Liveview, all bindings are handled with it.